File: Login/app/src/main/java/com/example/login/MainActivity.java
package com.example.login;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.view.ViewOutlineProvider;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button miButton = (Button) findViewById(R.id.button);
        miButton.setText("email");

        final TextView mitexto = (TextView) findViewById(R.id.button);





    }
}

File: PrimerParcial/app/src/main/java/com/example/primerparcial/MainActivity.kt
package com.example.primerparcial

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_login.*
import kotlinx.android.synthetic.main.activity_login.view.*
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_quien_soy.*


class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


            BtnIr.setOnClickListener {
                val intentUno: Intent = Intent(this, login::class.java)
                startActivity(intentUno)
                finish()

                        BtnAtrasLogin.setOnClickListener {
                            BtnAtrasLogin
                            finish()
                        }


                    }
                }
            }
        }
    }

